examples/graph_generation_examples/hierarchical_graph_example.py

```
@@ -0,0 +1,102 @@
+        if level == "atom" and "atomic_num" in graph.ndata:
+            atom_nums = graph.ndata["atomic_num"].tolist()
+            atom_types = [Chem.GetPeriodicTable().GetElementSymbol(int(num)) for num in atom_nums]
+            print(f"  - Atoms: {atom_types}")
+
+    # Visualize the graphs if requested
+    if args.visualize:
+        print("\nVisualizing hierarchical graphs...")
+        vis_dir = os.path.join(args.output_dir, "visualizations")
+        os.makedirs(vis_dir, exist_ok=True)
+
+        for level, path in graph_paths.items():
```

Ruff (0.11.9)

Loop control variable `level` not used within loop body

(B007)